src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -110,10 +109,17 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
     const headers = {
       Authorization: `Bearer ${authToken}`,
     };
+
+    // reqConfig might include the cache config from getMap, which could cache instances/${this.instanceId}/layers
+    // we do not want this as layer updates will not invalidate this cache, so we rathger cache to memory
+    const reqConfigWithMemoryCache = {
+      ...reqConfig,
+      cache: CACHE_CONFIG_30MIN_MEMORY,
```

Comment:
What if caching is disabled in `reqConfig` by setting `expiresIn` to 0 ? This will now override that setting and cache it in memory for 30mins. Not sure if this is expected behaviour.

Reply:
Good point, and I am not sure what the best approach here would be. The information needed from `fetchLayerParamsFromSHServiceV3` is stored in the layer's instance, so `fetchLayerParamsFromSHServiceV3` could only called if a new instance is created.

Do you think removing cache here is a viable option, or should we only override cache if cache was set in `reqConfig` unless it is `expiresIn: 0`?

Reply:
Yes, I think if cache is disabled for request, I wouldn't override it in `fetchLayerParamsFromSHServiceV3`.

Reply:
Made the changes so that `fetchLayerParamsFromSHServiceV3` will not be cached if caching is disabled by setting `expiresIn: 0`